geoshape/README.md

```diff
@@ -0,0 +1,28 @@
+## Geoshape track
+
+This track is based on [PlanetOSM](http://wiki.openstreetmap.org/wiki/Planet.osm) data.
+
+### Example Document
+
+```json
+{
+  "shape": "LINESTRING(-1.8212114 52.5538901, -1.8205573 52.554324)"
+}
+```
+
+### Parameters
+
+This track allows to overwrite the following parameters with Rally 0.8.0+ using `--track-params`:
+
+* `bulk_size` (default: 500)
```

Comment:
This differs from the value in the track (100) + also I am not sure whether the same bulk size for all corpora is right. I wonder whether we need to introduce dedicated parameters per corpus?